compiler/src/typed/typedtree.rei

```diff
@@ -41,7 +41,7 @@ type provide_flag =
 type rec_flag = Asttypes.rec_flag = | Nonrecursive | Recursive;
 type mut_flag = Asttypes.mut_flag = | Mutable | Immutable;
 type argument_label =
-  Asttypes.argument_label =
+  Parsetree.argument_label =
     | Unlabeled | Labeled(loc(string)) | Default(loc(string));
```

Comment:
This looks to be the alias, so if you do the Asttypes aliasing in parsetree, you don't need to change anything in this directory